config/public.php

```diff
@@ -133,6 +133,14 @@
 
         'redirect_default' => (bool) env('LOCALIZATION_REDIRECT_DEFAULT', false),
 
+        /*
+         * This option defines the default localization, when used, the localization parameter will be removed from the URL.
+         *
+         * Applies when using the `localizedRoute` helper.
+         */
+
+        'hide_default' => env('LOCALIZATION_HIDE_DEFAULT'),
```

Comment:
Need strict typing and default value.

```suggestion
        'hide_default' => (bool) env('LOCALIZATION_HIDE_DEFAULT', false),
```